src/NotificationsApp.vue

```diff
@@ -49,6 +49,8 @@
 							:index="2016" />
 						<Notification v-for="(n, index) in notifications"
 							:key="n.notificationId"
+							:object-type="n.objectType"
+							:app="n.app"
```

Comment:
This is already populated with the `v-bind="n"` below.

When you removed it, you can rebuild again.
Afterwards (if you are familiar with git) you could squash all the commits into 1 (so we don't have 8 commits to change back and forth 3 lines in the end)

Reply:
Makes sense, I fix, squash and force push

Reply:
Fixed @nickvergessen